test/test_kohlschuetter.py

```diff
@@ -1,58 +1,62 @@
 import io
-from lxml import etree
 import os
 import re
-import unittest
 
 import numpy as np
+import pytest
+from lxml import etree
 
 from dragnet import Blockifier, BlockifyError
 from dragnet.features import KohlschuetterFeatures
 from dragnet.compat import range_, str_cast
 
 
-with io.open(os.path.join('test', 'datafiles', 'HTML', 'page_for_testing.html'), 'r') as f:
-    big_html_doc = f.read()
-
-
-class KohlschuetterUnitBase(unittest.TestCase):
+@pytest.fixture(scope="module")
+def html():
+    fname = os.path.join("test", "datafiles", "HTML", "page_for_testing.html")
+    with io.open(fname, mode="rt") as f:
+        html_ = f.read()
+    return html_
 
-    def block_output_tokens(self, blocks, true_tokens):
-        """blocks = the output from blockify
-           true_tokens = a list of true tokens"""
 
-        self.assertTrue(len(blocks) == len(true_tokens))
+def block_output_tokens(blocks, true_tokens):
+    """
+    blocks = the output from blockify
+    true_tokens = a list of true tokens
+    """
+    assert len(blocks) == len(true_tokens)
+    for k in range_(len(blocks)):
+        block_tokens = re.split(r"\s+", blocks[k].text.strip())
+        assert block_tokens == true_tokens[k]
 
-        for k in range_(len(blocks)):
-            block_tokens = re.split('\s+', blocks[k].text.strip())
-            self.assertEqual(block_tokens, true_tokens[k])
 
-    def link_output_tokens(self, blocks, true_tokens):
-        self.assertTrue(len(blocks) == len(true_tokens))
+def link_output_tokens(blocks, true_tokens):
+    assert len(blocks) == len(true_tokens)
+    link_tokens = [ele.link_tokens for ele in blocks]
+    for k in range_(len(link_tokens)):
+        assert link_tokens[k] == true_tokens[k]
 
-        link_tokens = [ele.link_tokens for ele in blocks]
-        for k in range_(len(link_tokens)):
-            self.assertEqual(link_tokens[k], true_tokens[k])
 
-    def css_output_tokens(self, blocks, attrib, true_tokens):
-        self.assertEqual(len(blocks), len(true_tokens))
-        for k in range_(len(blocks)):
-            css_tokens = re.split('\s+', blocks[k].css[attrib].strip())
-            self.assertEqual(css_tokens, true_tokens[k])
+def css_output_tokens(blocks, attrib, true_tokens):
+    assert len(blocks) == len(true_tokens)
+    for k in range_(len(blocks)):
+        css_tokens = re.split(r"\s+", blocks[k].css[attrib].strip())
+        assert css_tokens == true_tokens[k]
 
 
-class TestBlockifier(KohlschuetterUnitBase):
+class TestBlockifier1(object):
```

Comment:
nit: is there a particular reason for the `1` in `TestBlockifier1`?

Reply:
Ah, that's a mistake — I named my versions of the unittest classes differently so I could run them in parallel with the originals, but forgot to rename these. Will fix.